app/femr/ui/views/medical/listVitals.scala.html

```diff
@@ -94,6 +99,11 @@
                         vitalMap.get("heightInches", vitalMap.getDate(dateIndex - 1)),
                         viewModel.getSettings.isMetric,
                         ""
+                    ) /@outputHeightOrNA(
+                        vitalMap.get("heightMeters", vitalMap.getDate(dateIndex - 1)),
+                        vitalMap.get("heightCm", vitalMap.getDate(dateIndex - 1)),
+                        !viewModel.getSettings.isMetric,
+                        ""
                     )
```

Comment:
The way you manage this is perfect "heightMeters" always has the height value in meters and "heightFeet" always has the height value in feet